tests/rows/run.sh

```diff
@@ -17,10 +18,14 @@ run_sql "drop database if exists \`$DB_NAME\`;"
 # build data on mysql
 run_sql "create database $DB_NAME;"
 run_sql "create table $DB_NAME.$TABLE_NAME (id int not null auto_increment primary key, a varchar(24));"
+run_sql "create table $DB_NAME.$TABLE_NAME2 (id bigint unsigned not null auto_increment primary key, a varchar(24));"
 
-# insert 100 records
+# insert 102 records
 run_sql_file "$cur/data/rows.t.0.sql"
 
+# insert 102 records
+run_sql_file "$cur/data/rows.t2.0.sql"
+
```

Comment:
Can you put this test into another new integration test? This integration test will check the file count. The negative or bigint primary key will affect the result.

Reply:
OK. I will put it into a new integration test for checking the bigint and negative primary key.